Dockerfile

```diff
@@ -24,12 +28,10 @@ RUN apt-get install -y git jq yarn && \
     apt-get purge -y git jq yarn && \
     apt-get autoremove --yes
 
-RUN adduser --uid 9000 --gecos "" --disabled-password app
-COPY . ./
-RUN chown -R app:app ./
 RUN chown -R app:app $PREFIX
 
 USER app
+COPY . ./
```

Comment:
Can you explain the reordering of the copy and chown lines? I think we might still want to copy and change permissions as root before dropping down to the `app` user.